src/gui/sdlmain.cpp

```diff
@@ -3244,6 +3245,49 @@ static void set_output(Section *sec, bool should_stretch_pixels)
 //extern void UI_Run(bool);
 void Restart(bool pressed);
 
+static void SetPriorities(const std::string_view foreground_pref,
+                          const std::string_view background_pref)
+{
+	// lamda to convert a string preference to priority level (enm)
```

Comment:
typo: lamda

Reply:
What's "enm"? According to Google, it could stand for "ethical non-monogamy" 🦜

Reply:
Lol :joy: 
(knocked it out.. self-documenting)